service/deployer/notifier/slack/slack.go

```diff
@@ -0,0 +1,165 @@
+package slack
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/nlopes/slack"
+
+	microerror "github.com/giantswarm/microkit/error"
+	micrologger "github.com/giantswarm/microkit/logger"
+
+	eventerspec "github.com/giantswarm/draughtsman/service/deployer/eventer/spec"
+	"github.com/giantswarm/draughtsman/service/deployer/notifier/spec"
+)
+
+const (
+	// goodColour is the colour to use for success Slack messages.
+	goodColour = "good"
+	// dangerColour is the colour to use for failure Slack messages.
+	dangerColour = "danger"
+
+	// titleFormat is the format for titles for Slack messages.
+	titleFormat = "%v - %v"
```

Comment:
I think this is rather obvious. For me it would be more helpful to know how the result of the title would look like and which information will be used to be substituted with the placeholders.